client/rest/package.json

```diff
@@ -7,9 +7,8 @@
     "start": "node src/index.js",
     "lint": "eslint .",
     "test": "mocha --full-trace --recursive ./test",
-    "coverage:test": "nyc --require mocha --reporter=lcov npm run test",
-    "lint:jenkins": "eslint -o lint.client.rest.xml -f junit . || exit 0",
-    "test:jenkins": "JUNIT_REPORT_PATH=tests.client.rest.xml mocha --reporter mocha-jenkins-reporter --forbid-only --full-trace --recursive ./test || exit 0"
+    "test:jenkins": "nyc --require mocha --reporter=lcov npm run test",
+    "lint:jenkins": "eslint -o lint.client.rest.xml -f junit . || exit 0"
```

Comment:
would prefer you keep ordering
lint
test
lint:jenkins
test:jenkins

Reply:
ok